### ISSUE TITLE
Add social links for community partners

### ISSUE THREAD

**Issue by bhansa** [HydPy/pyconfhyd2020#61]
As of now few community partners data has twitter links available and they should be added on the website.

**Comment by inovizz** [comment]
yes, let's add their community website link and if not then social profile link(twitter/linkedin/facebook/telegram)
